docs/03_linki.md

```diff
@@ -0,0 +1,44 @@
+# Społeczność, fora i czaty
+
+## Polska
+- (czat) przestrzeń MeshtasticPL na komunikatorze Matrix https://matrix.to/#/#meshtasticpl:matrix.org
+- Mapa (i serwer MQTT) https://loranet.pl
+- https://lora.waw.pl/
+  - forum https://lora.waw.pl/forum/
+- https://przejdznamediumfast.pl/
```

Comment:
Na razie nie mieszałbym nowym użytkownikom. Jak chcesz zostawić, to ewentualnie przenieś prosze na koniec strony, pod  "Artykuły różne."

Reply:
Przeniesione.